src/code/circuit_python/motor.md

```diff
@@ -0,0 +1,232 @@
+# Spin a Wheel
+
+In the previous tutorial we used some simple functions to turn a light
+on and off.  Lets use some more advanced code to now control a motor.
+In this program we'll make use of a
+[VEX MC-29 motor controller](https://www.vexrobotics.com/276-2193.html).
+These motor controllers use a
+[Pulse Width Modulation (PWM)](https://en.wikipedia.org/wiki/Pulse-width_modulation)
+signal to determine direction and magnitude of the motor rotation they
+control.
+
+For this program, we're going to use two new modules: `pwmio` and
+`adafruit_motor`.
+
+## The pwmio Module
+
+PWM is a simple way to send an analog signal over a digital pin. It is
+commonly used for controlling motors, servos, and other actuators. The
+`pwmio` module provides the `PWMOut` class to efficiently generate PWM
+signals on a pin using our microcontroller's built-in timers. There are
+two important values for us to control with a `PWMOut` object: frequency
+and duty cycle.
+
+Frequency is the number of times each second that the signal changes from
+low to high and back to low. This is measured in Hertz (Hz).
```

Comment:
Add: "One Hertz is equal to one millisecond."

Reply:
Added a definition for Hertz. (One hertz is not one millisecond.)